events/app/views/partials/event_without_schedule.njk

```diff
@@ -111,13 +111,17 @@
                 {% componentJsx 'Button', {type: 'primary', disabled: true, text: 'Fully booked'} %}
               {% else %}
                 <div class="js-eventbrite-ticket-button" data-eventbrite-ticket-id="{{ event.eventbriteId }}">
-                  {% componentJsx 'Button', {
-                    type: 'primary',
-                    url: 'https://www.eventbrite.com/e/' + event.eventbriteId + '/',
-                    eventTracking: "{category: 'component', action: 'booking-tickets:click', label: 'event-page'}",
-                    text: 'Book free tickets',
-                    icon: 'ticket'
-                  } %}
+                {# We don't use primary button here as we need to set the js-eventbrite-ticket-button-text #}
```

Comment:
Wonder if it's worth replacing the button with the 'Searching for tickets…' text, so we can use the component?

Reply:
Not sure I get what you mean?
So have a button, swap out for text, swap out for button?

Reply:
I just removed the loading state actually.
It could get someone stuck in a crap state, so we only update when we have something to update with.